documentation/Changelog.md

```diff
@@ -35,7 +42,15 @@ This version of MSBuild will ship with Visual Studio 2019 version 16.11.0 and .N
 * Switched to OneLocBuild for localization PRs (#6561).
 * Moved to latest Ubuntu image for PR test legs (#6573).
 
-#### Documentation
+## MSBuild 16.10.2
+
+This version of MSBuild shipped with Visual Studio 2019 version 16.10.2 and will ship with .NET SDK 5.0.302.
```

Comment:
Same as line 4 comment, should be `is shipped` unless it is emphasizing on the past state of 16.10.2